Jakefile.js

```diff
@@ -727,7 +719,15 @@ compileFile(
     [builtLocalDirectory, copyright, builtLocalCompiler].concat(languageServiceLibrarySources).concat(libraryTargets),
     /*prefixes*/[copyright],
     /*useBuiltCompiler*/ true,
-    { noOutFile: false, generateDeclarations: true });
+    { noOutFile: false, generateDeclarations: true },
+            /*callback*/ function () {
+
+        prependFile(copyright, tsserverLibraryDefinitionFile);
+
+        // Appending 'export = ts;' at the end of the server library file to turn it into an external module
+        var tsserverLibraryDefinitionFileContents = fs.readFileSync(tsserverLibraryDefinitionFile).toString() + "\r\nexport = ts;";

```

Comment:
this will beak users who have been using this file as global. we should add an additional line `export as namespace ts;`